.goreleaser.yaml

```
@@ -89,6 +89,8 @@ dockers:
       - ghcr.io/dynatrace/dynatrace-otel-collector/dynatrace-otel-collector:latest-amd64
       - public.ecr.aws/dynatrace/dynatrace-otel-collector:{{ .Version }}-amd64
       - public.ecr.aws/dynatrace/dynatrace-otel-collector:latest-amd64
+      - docker.io/dynatrace/dynatrace-otel-collector:{{ .Version }}-amd64
+      - docker.io/dynatrace/dynatrace-otel-collector:latest-amd64
```

Dockerhub images are usually not prefixed at all. please also change that for the following occurrences
```suggestion
      - dynatrace/dynatrace-otel-collector:{{ .Version }}-amd64
      - dynatrace/dynatrace-otel-collector:latest-amd64
```